gapis/api/vulkan/api/renderpass_framebuffer.api

```diff
@@ -224,13 +224,155 @@ Thefore there is no need to repeat the KHR structs
+  )
+
+  inputAttachments := subpassDescription2.pInputAttachments[0:subpassDescription2.inputAttachmentCount]
+  for j in (0 .. subpassDescription2.inputAttachmentCount) {
+    attachmentReference2 := inputAttachments[j]
+    internalSubpassDescription.InputAttachments[j] = createAttachmentReference2Object(attachmentReference2)
+  }
+
+  colorAttachments := subpassDescription2.pColorAttachments[0:subpassDescription2.colorAttachmentCount]
+  for j in (0 .. subpassDescription2.colorAttachmentCount) {
+    attachmentReference2 := colorAttachments[j]
+      internalSubpassDescription.ColorAttachments[j] = createAttachmentReference2Object(attachmentReference2)
+  }
+
+  if subpassDescription2.pResolveAttachments != null {
+    resolveAttachments := subpassDescription2.pResolveAttachments[0:subpassDescription2.colorAttachmentCount]
+    for j in (0 .. subpassDescription2.colorAttachmentCount) {
+      attachmentReference2 := resolveAttachments[j]
+        internalSubpassDescription.ResolveAttachments[j] = createAttachmentReference2Object(attachmentReference2)
+    }
+  }
+
+  if (subpassDescription2.pDepthStencilAttachment != null) {
+    attachmentReference2 := subpassDescription2.pDepthStencilAttachment[0]
+    attachmentReference2Object := createAttachmentReference2Object(attachmentReference2)
+    internalSubpassDescription.DepthStencilAttachment = new!AttachmentReference(attachmentReference2Object)
+  }
+
+  preserveAttachments := subpassDescription2.pPreserveAttachments[0:subpassDescription2.preserveAttachmentCount]
+  for j in (0 .. subpassDescription2.preserveAttachmentCount) {
+    internalSubpassDescription.PreserveAttachments[j] = preserveAttachments[j]
+  }
+  return internalSubpassDescription
+}
+
+sub SubpassDependency createSubpassDependencyObject(VkSubpassDependency subpassDependency) {
+  internalSubpassDependency := SubpassDependency(
+      SrcSubpass: subpassDependency.srcSubpass,
+      DstSubpass: subpassDependency.dstSubpass,
+      SrcStageMask: subpassDependency.srcStageMask,
+      DstStageMask: subpassDependency.dstStageMask,
+      SrcAccessMask: subpassDependency.srcAccessMask,
+      DstAccessMask: subpassDependency.dstAccessMask,
+      DependencyFlags: subpassDependency.dependencyFlags,
+    )
+  return internalSubpassDependency
+}
+
+sub SubpassDependency createSubpassDependency2Object(VkSubpassDependency2 subpassDependency2) {
+  internalSubpassDependency := SubpassDependency(
+      SrcSubpass: subpassDependency2.srcSubpass,
+      DstSubpass: subpassDependency2.dstSubpass,
+      SrcStageMask: subpassDependency2.srcStageMask,
+      DstStageMask: subpassDependency2.dstStageMask,
+      SrcAccessMask: subpassDependency2.srcAccessMask,
+      DstAccessMask: subpassDependency2.dstAccessMask,
+      DependencyFlags: subpassDependency2.dependencyFlags,
+    )
+  return internalSubpassDependency
+}
+
+sub AttachmentReference createAttachmentReferenceObject(VkAttachmentReference attachmentReference) {
```

Comment:
We should use this function for the `depthStencilAttachment` at line 420. I cannot comment on that line, so I am commenting it here.

Reply:
Done, good catch!